src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/Wizard.vue

```diff
@@ -1,18 +1,14 @@
 <template>
   <slot name="header"></slot>
   <v-stepper v-model="wizardStore.step" min-height="100dvh" :alt-labels="true" :mobile="$vuetify.display.mobile">
-    <v-stepper-header>
-      <template v-for="(step, index) in Object.values(wizard.steps)" :key="step.stage">
-        <v-stepper-item
-          color="primary"
-          :step="wizardStore.step"
-          :value="index + 1"
-          :title="step.title"
-          :rules="wizardStore.step <= index + 1 ? [] : [() => wizardStore.validationState[step.stage]]"
-        ></v-stepper-item>
-        <v-divider v-if="index !== Object.values(wizard.steps).length - 1" :key="`divider-${index}`" />
-      </template>
-    </v-stepper-header>
+    <slot name="stepperHeader">
```

Comment:
Do we need `<slot name="stepperHeader">`? I believe `<v-stepper-header>` should do it.

Reply:
We need to allow a slot for the v-stepper-header. Since the Application.vue uses 
ValidationState to mark some steps as invalid or not. 

References doesn't have this concept. So I had to abstract that piece out to Application.Vue.

Reply:
Ah, you just needed to expose this in `Application.vue` — got it!